Azure/azure-cli
# [AppService] Bugfix #14857: Let users update webapp config even with access restriction

**Description**
Fixes #14857 

**Testing Guide**
- Create an App Service and a VNET which has one subnet at least.
- Set Access Restriction by the subnet to the App Service
- Execute `az login` using a service principal which has a contributor role of the App Service and doesn't have any role the VNET.
- Execute `az webapp config set --resource-group {} --name {} --always-on {}`

Currently see error: 
![image](https://user-images.githubusercontent.com/14339314/99301637-1d81f080-2803-11eb-857b-29687884a596.png)

User with permissions to webapp should be able to update webapp even if there are access restrictions by subnet where user does not have permissions to subnet

---

This checklist is used to make sure that common guidelines for a pull request are followed.
